README.md

```diff
@@ -1224,9 +1224,11 @@ To make a release do the following:
   sbt tut
 ```
 
-* Publish a cross-version signed package
+* Publish a cross-version signed package (no cross-version available for Scala Native)
 ```
-  sbt +publishSigned
+  sbt +squantsJVM/publishSigned
```

Comment:
That's a good point, breaking it will probably slow down travis a bit while warming up sbt